ceno_zkvm/src/scheme/prover.rs

```diff
@@ -52,13 +52,14 @@ impl<E: ExtensionField, PCS: PolynomialCommitmentScheme<E>> ZKVMProver<E, PCS> {
     }
 
     /// create proof for zkvm execution
-    #[tracing::instrument(skip_all, name = "ZKVM_create_proof")]
+    #[tracing::instrument(skip_all, name = "ZKVM_create_proof", fields(profiling_1))]
     pub fn create_proof(
         &self,
         witnesses: ZKVMWitnesses<E>,
         pi: PublicValues<u32>,
         mut transcript: Transcript<E>,
     ) -> Result<ZKVMProof<E, PCS>, ZKVMError> {
+        let span = entered_span!("commit_to_fixed_commit", profiling_1 = true);
```

Comment:
Instead of pairs of and `entered_span` and `exit_span` like this

```rust
let span = entered_span(some_name);
// [... some code ...]
exit_span!(span);
```
in most cases we can probably use a less error-prone mechanism:

```rust
with_span!(some_name) {
  // [... some code ...]
}
```

(That requires to quickly write a `with_span` macro.  The exact syntax is only a suggestion.)

Reply:
I get your point, but I dislike the effect on indentation. Often, `[... some code...]` is not an expression, but a very large block. Because it's not a clear-cut decision, I would defer it to a separate issue/PR.

Reply:
Yes, we can do that in a separate PR.